IFC4x3/Properties/f/FailureLoad_37Y$o07UP6fwQ09RpopCdC/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FailureLoad_37Yo07UP6fwQ09RpopCdC" Name="FailureLoad" UniqueId="c78bfc80-1de6-46a7-a680-25bcf2ccc9cc" PrimaryDataType="IfcForceMeasure" />
```

Comment:
Could this be reused in other properties describing "breaking load"? It seems quite generally defined.